lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -560,7 +560,7 @@ def code_to_exec(code=code, module=module, ctx=ctx, rerun_data=rerun_data):
                                 # removal information has reached the web app
                                 # (see https://github.com/streamlit/streamlit/issues/9080).
                                 if not rerun_data.is_auto_rerun:
-                                    raise RuntimeError(
+                                    _LOGGER.error(
```

Comment:
nit: can you add a comment here on what happens if an app runs into this? I assume the requested fragment rerun will just be ignored?

Reply:
Or maybe just add this info to the error message as well

Reply:
I have extended the comment above with a small preamble

Reply:
Oh I have just seen your second comment. Good idea, I have just updated the error message as well.